README.md

```diff
@@ -0,0 +1,37 @@
+<p>
+    <a href="https://crates.io/crates/near-token"><img src="https://img.shields.io/crates/dv/near-token?style=flat-square&logo=near&label=crates.io" alt="Crates.io (latest)"></a>
+    <a href="https://docs.rs/near-token/latest/near_token"><img src="https://img.shields.io/docsrs/near-token?style=flat-square" alt="Docs.rs"></a>
+    <img src="https://img.shields.io/badge/rustc-1.68%2B-lightgray.svg?style=flat-square" alt="Rust Version">
+</p>
+
+# near-token
+near-token is crate for work with tokens in near-protocol.
+
+The crate includes NearToken type and constructors for converting data as NearToken and as u128 type values.
+
+## near-token examples 
+```rust
+use near_token::NearToken;
+
+fn main() {
+    assert_eq!(NearToken::from_near(10).as_near(), 10);
+    assert_eq!(
+        NearToken::from_near(1),
+        NearToken::from_yoctonear(10u128.pow(24))
+    );
+    assert_eq!(NearToken::from_yoctonear(10u128.pow(24)).as_near(), 1);
+}
+```
+## NearToken information
+NEAR is used to price computation and storage on the NEAR infrastructure. The network charges transaction fees in NEAR to process changes and transactions.
+ 
+
+
+
+### License
+
+This project is licensed under the [MIT license] and [Apache-2.0 license].
+
+[MIT license]: https://github.com/near/near_token/blob/main/LICENSE-MIT
+[Apache-2.0 license]:  https://github.com/near/near_token/blob/main/LICENSE-APACHE
+[For more information]: https://wiki.near.org/getting-started/near-token/
```

Comment:
I don't see this link used anywhere on the page, and it is also giving me 404 page

Reply:
```suggestion
```